### PR TITLE
add assembler recipe for ic2 nuclear components

### PR DESCRIPTION
**Pull request by xTwi**
Adds Nuclear Reactors and Reactor Chambers to Assembling Machine.

### CODE REVIEW

**Comment by xTwi** [comment]
Is something done on my bucket list. https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/8837

**Comment by xTwi** [comment]
Requesting a balance pass, AKA some people look over the values for duration/eu per tick, since I've finished adding recipes.

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java [hunk 1]
```diff
@@ -2513,9 +2513,36 @@ public void run3() {
...
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_Utility.getIntegratedCircuit(22), GT_OreDictUnificator.get(OrePrefixes.plateDense, Materials.Lead, 2L), GT_OreDictUnificator.get(OrePrefixes.plateDense, Materials.Titanium, 2L), GT_ModHandler.getModItem("IC2", "blockReactorChamber", 3L), GT_OreDictUnificator.get(OrePrefixes.cableGt08, Materials.Platinum, 1L), GT_OreDictUnificator.get(OrePrefixes.circuit, Materials.Advanced, 1L)}, GT_Values.NF, GT_ModHandler.getModItem("IC2", "blockGenerator", 1L, 5), 400, 480);
+
+        //Heat Exchangers
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_Utility.getIntegratedCircuit(21), GT_OreDictUnificator.get(OrePrefixes.plate, Materials.Silver, 3L), GT_OreDictUnificator.get(OrePrefixes.plate, Materials.Copper, 1L), GT_OreDictUnificator.get(OrePrefixes.plate, Materials.Aluminium, 3L), GT_OreDictUnificator.get(OrePrefixes.circuit, Materials.Basic, 1L)}, GT_Values.NF, GT_ModHandler.getModItem("IC2", "reactorHeatSwitch", 1L, 1), 60, 32);
```
Comment (Ethryan) [inline src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:2523]: These and the recipes below use the absolute limit of an energy tier i would rather see that you reduced the energy cost so it isn’t a multiplier of 32.

30 max lv
120 max mv
480 max hv

Please check these values.

Comment (xTwi) [reply src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java]: Done, does that look good? What about duration?

Comment (Ethryan) [reply src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java]: That looks good

**Review by Ethryan — APPROVED** [review]

**Comment by Dream-Master** [comment]
@Ethryan did you test stuff ingame ?

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java [hunk 2]
```diff
@@ -2513,9 +2513,36 @@ public void run3() {
...
+
+        //Reactor Plating
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_Utility.getIntegratedCircuit(22), GT_OreDictUnificator.get(OrePrefixes.plateDense, Materials.Lead, 1L), GT_ModHandler.getModItem("IC2", "itemPartAlloy", 8L)}, GT_Values.NF, GT_ModHandler.getModItem("IC2", "reactorPlating", 1L), 100, 120);
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_Utility.getIntegratedCircuit(21), GT_OreDictUnificator.get(OrePrefixes.plateDense, Materials.Copper, 4L), GT_OreDictUnificator.get(OrePrefixes.plate, Materials.Copper, 2L), GT_OreDictUnificator.get(OrePrefixes.plate, Materials.Silver, 2L), GT_ModHandler.getModItem("IC2", "reactorPlating", 1L)}, GT_Values.NF, GT_ModHandler.getModItem("IC2", "reactorPlatingHeat", 1L), 100, 120);
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_Utility.getIntegratedCircuit(23), GT_OreDictUnificator.get(OrePrefixes.plateDense, Materials.Lead, 4L), GT_ModHandler.getModItem("IC2", "itemPartAlloy", 4L), GT_ModHandler.getModItem("IC2", "reactorPlating", 1L)}, GT_Values.NF, GT_ModHandler.getModItem("IC2", "reactorPlatingExplosive", 1L), 100, 120);
+
+        //LV and MV Energy Hatches
```
Comment (Dream-Master) [inline src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:2542]: if you put lv and mv energy to code you should add the rest as well.

Comment (xTwi) [reply src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:2542]: The HV hatch and higher already have assembler recipes and can't be crafted without them. The MV and LV don't have those recipes, so I added them.

**Pull request merged**